k2/csrc/ragged_ops.h

```diff
@@ -1003,6 +1031,12 @@ Ragged<T> Stack(int32_t axis, int32_t num_srcs, Ragged<T> *src,
     @param [in] src  The ragged tensor to be unstacked.
     @param [in] axis  The axis to be removed, all the elements of this axis will
                       be rearranged into output Raggeds.
+    @param [in] empty_pos  Before unstack, we will pad the sublists along axis
+                  `axis` to the same size with empty lists(we don't actually do
+                  that, but think about it), `empty_pos` tells where to put
```

Comment:
Instead of `(we don't actually do
                  that, but think about it) `, could say: ` Before unstack, we will (conceptually) pad `...

Reply:
... also, it's not very standard in C++ to use strings as flags.  I think having a boolean "pad_right" might be more conventional.